RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -329,34 +344,34 @@ public void onClick(View v) {
     public void onDelete(final Project project) {
         AlertDialog.Builder builder = new AlertDialog.Builder(this);
         builder
-            .setTitle("Delete Project")
-            .setMessage("Deleting this project will remove all associated verse and chunk " +
-                "recordings.\n\nAre you sure?")
-            .setPositiveButton("Yes", new DialogInterface.OnClickListener() {
-                @Override
-                public void onClick(DialogInterface dialog, int which) {
-                    if (which == dialog.BUTTON_POSITIVE) {
-                        Project.deleteProject(mCtx, project);
-                        populateProjectList();
-                        hideProjectsIfEmpty(mAdapter.getCount());
-                        removeProjectFromPreferences(project);
-                        mNumProjects--;
-                        initializeViews();
+                .setTitle("Delete Project")
+                .setMessage("Deleting this project will remove all associated verse and chunk " +
+                        "recordings.\n\nAre you sure?")
+                .setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        if (which == dialog.BUTTON_POSITIVE) {
+                            Project.deleteProject(ActivityProjectManager.this, project);
+                            populateProjectList();
+                            hideProjectsIfEmpty(mAdapter.getCount());
+                            removeProjectFromPreferences(project);
+                            mNumProjects--;
+                            initializeViews();
+                        }
                     }
-                }
-            })
-            .setNegativeButton("No", new DialogInterface.OnClickListener() {
-                @Override
-                public void onClick(DialogInterface dialog, int which) {
-                    dialog.dismiss();
-                }
-            });
+                })
+                .setNegativeButton("No", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        dialog.dismiss();
+                    }
+                });
 
         AlertDialog dialog = builder.create();
         dialog.show();
     }
 
-    public void exportProgress(int progress){
+    public void exportProgress(int progress) {
```

Comment:
An export refactor needs to occur in which it makes use of the TaskFragment. I would have done this for this merge, however it should encompass fixing AmazonS3's upload, which should include Door43 requirements, as well as actually making modifications to the sharing list. See Issue #642